examples/src/main/scala/io/finch/iteratee/Main.scala

```diff
@@ -11,15 +9,14 @@ import io.finch.catsEffect._
 import io.finch.circe._
 import io.iteratee.{Enumerator, Iteratee}
 
-/**
-  * A Finch application featuring iteratee-based streaming support.
-  * This approach is more advanced and performant then basic [[com.twitter.concurrent.AsyncStream]]
+import scala.util.Random
+
+/** A Finch application featuring iteratee-based streaming support. This approach is more advanced and performant then basic
+  * [[com.twitter.concurrent.AsyncStream]]
   *
   * There are three endpoints in this example:
   *
-  *  1. `sumJson` - streaming request
-  *  2. `streamJson` - streaming response
-  *  3. `isPrime` - end-to-end (request - response) streaming
+  *   1. `sumJson` - streaming request 2. `streamJson` - streaming response 3. `isPrime` - end-to-end (request - response) streaming
```

Comment:
The formatting got a bit messed up here - it's supposed to be a numbered list which can be achieved by just using `1.`
```suggestion
  *   1. `sumJson` - streaming request
  *   1. `streamJson` - streaming response
  *   1. `isPrime` - end-to-end (request - response) streaming
```